invisible_cities/reco/corrections_test.py

```diff
@@ -268,17 +269,34 @@ def test_apply_all_correction_single_maps_raises_exception_when_invalid_map(map_
 
 @few_examples
 @given(float_arrays(size      = 1,
-                   min_value = -198,
-                   max_value = +198),
+                    min_value = -198,
+                    max_value = +198),
        float_arrays(size      = 1,
-                   min_value = -198,
-                   max_value = +198),
+                    min_value = -198,
+                    max_value = +198))
+def test_apply_geo_correction_properly(map_filename, x, y):
+    """
+    Due the map taken as input, the geometric correction
+    factor must be 1.
+    """
+    maps      = read_maps(map_filename)
+    load_corr = apply_geo_correction(map_e0 = maps, norm_strat = NormStrategy.max)
```

Comment:
```suggestion
    load_corr = apply_geo_correction(map_e0=maps, norm_strat=NormStrategy.max)
```
When keyword arguments are given in the same line, it's usually preferred to remove the whitespaces around the `=`